### PR TITLE
Updates to 'setup.py' for the 0.21 release

### PR DESCRIPTION
**Pull request by pwt**
- Update the supported Python versions
- Change `Development Status` to `4 - Beta`
- Apply Black formatting
- Add python_requires=">=3.5"

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -13,63 +13,64 @@ class PyTest(TestCommand):
...
-    'Topic :: Software Development :: Libraries :: Python Modules',
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3.5",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
```
Comment (hugovk) [inline setup.py:56]: Optionally make it more explicit:
```suggestion
    "Programming Language :: Python :: 3.9",
    "Programming Language :: Python :: 3 :: Only",
```

**Review by hugovk — APPROVED** [review]
Looks good 👍

File: setup.py (hunk 1 above)
Comment (pwt) [reply setup.py:56]: Thanks, done.

**Comment by pwt** [comment]
Tested against TestPyPI, all good, so I'm going to merge this.

**Pull request merged**